src/backend/aspen/app/views/sample.py

```diff
@@ -150,57 +149,39 @@ def _format_lineage(sample: Sample) -> dict[str, Any]:
 @application.route("/api/sequences", methods=["GET"])
 @requires_auth
 def prepare_sequences_download():
```

Comment:
@shanaxel42 I think github wasn't expanding this by default.

Reply:
AH MY BAD, sorry about that